__tests__/app.test.js

```diff
@@ -41,4 +41,39 @@ describe('/api/topics', () => {
         return request(app).get('/api/not_a_route')
         .expect(404)
     })
+})
+
+describe('/api/articles/:article_id', () => {
+    test('GET:200 sends one article object to the client', () => {
+        return request(app).get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body
+            const [ articleObj ] = article
+
+            expect(articleObj.article_id).toBe(1)
+            
+            expect(Object.keys(articleObj)).toEqual(['article_id', 'title', 'topic', 'author', 'body', 'created_at', 'votes', 'article_img_url'])

```

Comment:
I would suggest using a flexible matcher eg expect.objectContaining or .toMatchObject. - these mean if changes are made the test will still pass later on